tendermint/src/lite.rs

```diff
@@ -1,6 +0,0 @@
-//! Core logic and traits of a light client.
```

Comment:
This file isn't used by any modules in the tendermint crate.

Reply:
:clap: